src/TTextEdit.cpp

```diff
@@ -1268,10 +1268,16 @@ void TTextEdit::mousePressEvent(QMouseEvent* event)
         popup->addAction(action3);
 
         if (mPA != mPB && mpHost->mEnableTextAnalyzer) {
-            mpContextMenuAnalyser = new QAction("Analyse characters", this);
-            connect(mpContextMenuAnalyser, SIGNAL(hovered()), this, SLOT(slot_analyseSelection()));
-            mpContextMenuAnalyser->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
-                                                 .arg(tr("Hover on this item to display the Unicode codepoints in the <i>first line</i> of the selection.")));
+            mpContextMenuAnalyser = new QAction(tr("Analyse characters"), this);
+            // NOTE: If running inside the Qt Creator IDE using the debugger with
+            // the hovered() signal can be *problematic* - as hitting a
+            // breakpoint - or getting an OS signal (like a Segment Violation)
+            // can hang not only Mudlet but also Qt Creator and possibly even
+            // your Desktop - though for *nix users swithing to a console and
+            // killing the gdb debugger instance run by Qt Creator will restore
+            // normality.
```

Comment:
TMI

Reply:
Very nasty if it happens though. I feel obliged to warn others to not get caught out like I was - it is *no fun* when **your *whole OS* seems to hang**!